app/Community/Components/ActivePlayers.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Community\Components;
+
+use App\Community\Services\ActivePlayersService;
+use Illuminate\Contracts\View\View;
+use Illuminate\Http\Request;
+use Illuminate\View\Component;
+
+class ActivePlayers extends Component
+{
+    protected ActivePlayersService $activePlayersService;
+    public ?string $initialSearch = null;
+
+    public function __construct(ActivePlayersService $activePlayersService, Request $request)
```

Comment:
```php
public function __construct(
    protected ActivePlayersService $activePlayersService, 
    Request $request
)
```
defining visibility with the parameter allows to skip the member definition and assignment.

Reply:
Thanks, resolved in latest.